tsl/src/bgw_policy/reorder_api.c

```diff
@@ -260,8 +260,10 @@ policy_reorder_add(PG_FUNCTION_ARGS)
 										&check_name,
 										&owner,
 										true,
+										false,
```

Comment:
Nit: for boolean parameters it is usually good to add a comment with the name of the parameter. It makes it easier to maintain the code:
```suggestion
										false, /* fixed_schedule */
```